modules/ibi-installer-installation-config.adoc

```diff
@@ -34,7 +34,13 @@ If the seed image requires a separate private registry authentication, add the a
 |Specification|Type|Description
 |`shutdown`|`boolean`|Specifies if the host shuts down after the installation process completes. The default value is `false`.
 |`extraPartitionStart`|`string`|Specifies the start of the extra partition used for `/var/lib/containers`. The default value is `-40Gb`, which means that the partition will be exactly 40Gb in size and uses the space 40Gb from the end of the disk. If you specify a positive value, the partition will start at that position of the disk and extend to the end of the disk.
-|`extraPartitionLabel`|`string`|The label of the extra partition you use for `/var/lib/containers`. The default label is `varlibcontainers`.
+|`extraPartitionLabel`|`string`|The label of the extra partition you use for `/var/lib/containers`.
+
+[NOTE]
+====
+The default partition label for the installation ISO is `varlibcontainers`. You must ensure that the partition label in the installation ISO matches the partition label set in the machine configuration for the seed image. If the partition labels are different, the partition mount fails during installation on the host. For more information, see "Configuring a shared container partition between ostree stateroots".
```

Comment:
I like the new wording, but I would move the description of the default value to the previous table

Reply:
Done, thanks